backend/config/settings.py

```diff
@@ -409,6 +409,7 @@
 ]
 
 env_base_url = env.str("DJANGO_BASE_URL", "")
+login_client_id = secret("LOGIN_CLIENT_ID", "")
```

Comment:
```suggestion
login_client_id = secret("LOGIN_CLIENT_ID", "")
```

Per discussion, this value is obtained from $VCAP_SERVICES for the app, like $DJANGO_SECRET_LOGIN_KEY and will need the secret() in order to grab the value